db/migrations/sql/1_initial_schema.sql

```diff
@@ -652,8 +652,8 @@ DO
 
 $$
 BEGIN
-    IF NOT "table_exists"('ar_internal_metadata') THEN
-        CREATE TABLE "schema_migrations" (
+    IF NOT "table_exists"('schema_migrations') THEN
```

Comment:
@MikelAlejoBR Should `1_initial_schema.sql` reflect original state as rails app had ? if so should we create `ar_internal_metadata` here ? (because in migration here we are deleting `ar_internal_metadata `)

Reply:
I don't think so, because just above this code block we're checking for the `ar_internal_metadata` table. However I was checking here for `ar_internal_metadata` again instead of `schema_migrations`... Maybe this belongs to another PR